tester_app/client/app/bundles/HelloWorld/containers/HelloWorldContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import HelloWorld from '../components/HelloWorld';
+
+// Simple example of a React component
+export default class HelloWorldContainer extends React.Component {
+  static displayName = 'HelloWorldContainer';
+
+  static propTypes = {
+    initialName: PropTypes.string.isRequired, // this is passed from the Rails view
+  }
+
+  state = {name: this.props.initialName} // how to set initial state in es2015 class syntax
+
+  updateName(name) {
+    this.setState({name: name});
+  }
+
+  render() {
+    const updateName = ::this.updateName;
+    const name = this.state.name;
```

Comment:
@jbhatab Our suggestions are "opinionated". They are going to be a bit like coffee. Might tast bad at first, and then you'll like it!